app/views/bulk_discounts/index.html.erb

```diff
@@ -0,0 +1,15 @@
+
+<h1> Bulk Discounts </h1>
+
+<ul>
+
+  <% @bulk_discounts.each do |bulk_discount| %> 
+ 
+  <section id ="bulk_show_link-<%= bulk_discount.id %>">
+    <li><%= "Bulk Discounts: " %><%= link_to " #{bulk_discount.id}", "/merchants/#{bulk_discount.merchant_id}/bulk_discounts/#{bulk_discount.id}" %></li>  
```

Comment:
This bulk_discount.merchant_id tripped me up for a second, but makes sense why and how it's referencing itself.